### PR TITLE
Feature create story scroll mode

### LINKED ISSUE #32: Create add story page
A normal user (not superuser) can add a story in scroll mode (just plain text)

### PR DESCRIPTION
**Pull request by KOngTNP**
fix create_story problem that cannot upload image.
-Can create story.
-Can upload the image.
-The story show on homepage.

### CODE REVIEW

File: TELLING/forms.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Story
+
+class StoryForm(forms.ModelForm):
+    class Meta:
+        model= Story
+        fields= ("title","content", "author", "categories", "thumbnail",)
```
Comment (bluaries) [inline TELLING/forms.py:7]: I add title box to this form in my branch too but it's ok.

**Review by bluaries — APPROVED** [review]
It's working. let's merge it to master.

**Review by mark47546 — APPROVED** [review]
Well done. Its working pls merge.

**Review by theethaj — APPROVED** [review]
It's okay, you can merge.

**Pull request merged**